lib/elixir/lib/agent.ex

```diff
@@ -78,13 +78,13 @@ defmodule Agent do
     * `:id` - the child specification identifier, defaults to the current module
     * `:start` - how to start the child process (defaults to calling `__MODULE__.start_link/1`)
     * `:restart` - when the child should be restarted, defaults to `:permanent`
-    * `:shutdown` - how to shut down the child
+    * `:shutdown` - the shut down strategy
```

Comment:
Using the word strategy is confusing, because we also have the supervision strategy. I think we can try to be more descriptive here.

Reply:
my change is because "how to" sound weird when we are talking about a `timeout`. We need to come up with a way that covers `milliseconds`, `:infinity` and `:brutal_kill`